assignments/admin.py

```diff
@@ -1,18 +1,51 @@
 from django.contrib import admin
-from assignments.models import Assignment, Section, OpenTextTask
-from nested_admin.nested import NestedModelAdminMixin, NestedStackedInline, NestedTabularInline
+from assignments.models import (Assignment, Section, OpenTextTask, BudgetTextTask, BudgetTextTarget, MapObject,
+                                BudgetMapTask)
+from nested_admin.nested import NestedModelAdminMixin, NestedTabularInline
 from leaflet.admin import LeafletGeoAdmin
+from django import forms
+
+
+class BudgetForm(forms.ModelForm):
+    def clean_range(self):
+        '''
+        Include upper boundary if boundary value is set
+        :return: numericrange object with upper boundary included
+        :rtype: NumericRange
+        '''
+        if self.cleaned_data['range'] is not None:
+            self.cleaned_data['range']._bounds = '[]'
```

Comment:
Accessing the `_bounds` like that seems a bit alarming. There isn't cleaner way to do this?